types/fg-loadcss/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for fg-loadcss 3.1
+// Project: https://github.com/filamentgroup/loadCSS
+// Definitions by: Noah Overcash <https://github.com/ncovercash>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function loadCSS(href: string, before?: HTMLElement, media?: string, attributes?: Record<string, string>): HTMLLinkElement;
```

Comment:
AFAIK, this seems to be UMD module, so this one would require minor change:
https://unpkg.com/fg-loadcss@3.1.0/src/loadCSS.js
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#identifying-a-umd-library
https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html#examples-of-umd-libraries

```ts
export as namespace loadCSS ...

declare function loadCSS...

export = loadCSS;
````